android/.gitignore

```diff
@@ -1,40 +1,23 @@
-bin
-gen
-/libs/
-.settings
-resbuilder
-spoon-output
-obj
-local
+# default android studio ignore list
+*.iml
 .gradle
-build
-3rd_party/api-android/
-RELEASE
-.idea
+/local.properties
+/.idea/caches
+/.idea/libraries
+/.idea/modules.xml
+/.idea/workspace.xml
+/.idea/navEditor.xml
+/.idea/assetWizardSettings.xml
```

Comment:
Why not the whole .idea ?

Reply:
This is the default .gitignore generated by android studio. My guess is that it allows to keep custom run configurations